### PR TITLE
resource: mark torpid nodes down for scheduling instead of draining them

### PR DESCRIPTION
**Pull request by garlick**
Problem: when a large cluster has a slow rank 0 node or experiences network problems, many nodes can become drained if they are unresponsive long enough to be declared torpid.  System administrators then have to manually undrain them.

Since it seems in practice, torpidity is a transient phenomena, change the resource module to mark torpid nodes "down" for scheduling then "up" if/when they again become responsive.  They are no longer drained.  They are logged as before, but other than that there is no permanent record of torpidity over time.

`flux resource list` will show these nodes as down.

WIP because `flux resource status` IMHO should show a new `torpid` line and subtract the torpid node count from the "avail" count.

### CODE REVIEW

**Comment by garlick** [comment]
Oops - I had forgotten to update `flux resource status` json test inputs with the new `torpid` field after the python commit.  Fixed that and also added a commit to update the admin guide.

**Comment by garlick** [comment]
Thanks @grondo for contributing the `flux-resource` changes and tests!  I have reviewed them and they look great to me.

File: src/modules/resource/monitor.c [hunk 1]
```diff
@@ -264,30 +264,38 @@ struct monitor *monitor_create (struct resource_ctx *ctx,
...
     if (monitor->size < inventory_size)
         monitor->size = inventory_size;
+
     if (flux_msg_handler_addvec (ctx->h, htab, monitor, &monitor->handlers) < 0)
         goto error;
+
+    /* Monitor currently doesn't do anything on follower ranks,
+     * except respond to RPCs with a human readable error.
+     */
+    if (ctx->rank > 0)
```
Comment (grondo) [inline src/modules/resource/monitor.c:333]: Nice cleanup!

File: src/modules/resource/monitor.c [hunk 2]
```diff
@@ -169,7 +176,7 @@ static void broker_online_cb (flux_future_t *f, void *arg)
     if (!(up = group_get (f))) {
         flux_log (h,
                   LOG_ERR,
-                  "monitor: group.get: %s",
+                  "monitor: broker.online: %s",
```
Comment (grondo) [inline src/modules/resource/monitor.c]: This seems like it was meant for a different commit? (Is now in `resource: monitor broker.torpid `)

File: doc/man1/flux-resource.rst [hunk 3]
```diff
@@ -70,7 +70,7 @@ Show the scheduler view of resources.
 
 In the scheduler view, excluded resources are always omitted, and
 unavailable resources are shown in the "down" state regardless of the
-reason (drained, offline, etc).  Valid states in the scheduler view are:
+reason (drained, offline, torpid, etc).  Valid states in the scheduler view are:
```
Comment (grondo) [inline doc/man1/flux-resource.rst:73]: Commit message typo: pgae

**Review by grondo — APPROVED** [review]
This looks great to me! Just some very minor comments. Feel free to squash some of the incremental work if you'd like or leave it as is.

**Comment by garlick** [comment]
Thanks!  I addressed your comments and rebased on current master.  I'll set MWP.

**Pull request merged**